src/System.Runtime/tests/TestLoadAssembly/ClassesSample.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+public class PublicClassSample
+{
+    public PublicClassSample() { }
+    public PublicClassSample(int param) { }
+}
+
+class PrivateClassSample
```

Comment:
add access modifier

Reply:
Elements defined in a namespace cannot be explicitly declared as private, protected, or protected internal

Reply:
so it is a internal or a public class ?

Reply:
```
.class private auto ansi beforefieldinit ConsoleApp1.Test
       extends [System.Runtime]System.Object
{
} // end of class ConsoleApp1.Test

```

Reply:
Okay Thanks for the info :)